### DISCUSSION TITLE
Ventilation Control

### DISCUSSION THREAD

**Discussion by Paulbhyo** [PeteRager/lennoxs30#186] [General]
Hello,  I wasn't sure where to ask this question.  Is there a setting or a variable that can be set to turn the ventilation on or off?   My system has a ventilator that is controlled by the ACC1 on the S30 Hub V2.

**Comment by PeteRager** [comment]
There should be a switch that gets created to open / close the ventilation damper.  Check to see if that is being created.  It should be attached to the S30 device.  If it is not there, we’ll need to look at your S30 config to see how to enable it.

**Comment by Paulbhyo** [comment]
I don’t see anything under the developer tools tab for ventilator or damper in the states.   I don’t think the operation of the ventilation would change if I re-configured the unit to be a damper..  I’ll need to tinker with the Lennox settings this weekend and see.   If I do change the settings would I need to refresh the integration or anything, or should it pick it up?

**Comment by PeteRager** [comment]
Let’s not change anything on your system.  I’ve never seen a system setup like yours and hence the integration is not coded to support this.  So, we’ll need to capture some information and then I should be able to update to support.   

We’ll need to capture the config and messages from your system.

Follow these instructions to enable message logging:

https://github.com/PeteRager/lennoxs30#reporting-bugs

Then reload the integration or restart HA
Wait a minute or two for everything to come up
From your S30 panel, toggle the ventilation (if it is on turn it off, if it’s off turn it on)
Wait a minute
Toggle it again
Wait a minute
Toggle it again

Let me know the order of the toggles (e,g on,off,on or off,on,off) Post the message log.  It’ll be 100kb or so.  

A photo of what you see in the panel would be helpful also, so that as I create the HA entities I can use a similar naming

**Comment by Paulbhyo** [comment]
I tried out the updated software and it looks like it is working pretty good.  at first I didn't understand how it functions.  I thought I had to enter a time and then hit the on switch, but it turns out when you use the numeric input in lovelace (via android) I had to click off the cell for the value to be applied and that started the ventilator.  ..    you might have it documented someplace, but it just needs the time to be set to turn it on.   turning it off zeroed the time.  

I found on my phone I could enter a time and then hit the switch,  and what that did was send the time to the thermostat and turn the ventilation to the 'Always on' mode.   I eventually figured it out.

**Comment by PeteRager** [comment]
That is some great feedback.  I’ll clarify the docs.  Initially in our conversation I did suggest that you’d set the number then turn on the switch.  

The number controls do behave differently on different devices.  On the web browse  using the arrow up control seems to trigger it.  

In a dashboard you can add a button that then calls the number service to set the entity to a value.  So you could create a ventilate 10 minute button, etc.
